<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarnCfg</class>
 <widget class="QWidget" name="WarnCfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Notification Rules</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="trafficTypeLabel">
          <property name="text">
           <string>Traffic type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="trafficType">
          <item>
           <property name="text">
            <string>Peak</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Offpeak</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Peak and offpeak</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Traffic direction:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="trafficDirection">
          <item>
           <property name="text">
            <string>Incoming</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Outgoing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Incoming and outgoing</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Notify when traffic exceeds:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="warnPer">
              <property name="text">
               <string>within:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_12">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="KDoubleNumInput" name="threshold">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="whatsThis">
               <string>When traffic for a month or billing period exceeds this limit, KNemo will emit a notification. The notification will appear once per session.</string>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="specialValueText">
               <string/>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="trafficUnits">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="KIntSpinBox" name="periodCount">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="periodUnits">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Notification Text</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="customTextCheck">
        <property name="text">
         <string>Custom notification text</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KTextEdit" name="customTextEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="legend">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>trafficType</tabstop>
  <tabstop>trafficDirection</tabstop>
  <tabstop>trafficUnits</tabstop>
  <tabstop>periodCount</tabstop>
  <tabstop>periodUnits</tabstop>
  <tabstop>customTextCheck</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>customTextCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>legend</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customTextCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>customTextEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
